### PR TITLE
Help: closes GROK-12025:  Update the Quality Assurance doc

### PR DESCRIPTION
**Pull request by ahadzhanian**

### CODE REVIEW

File: help/develop/how-to/report-tickets.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+  * **Attachment (optional)**. You are welcome to attach screenshots, GIFs,
+  recorded videos, etc. to complete the issue description.
+* **Due date**. Set the due date according to the priority, client’s request, or
+corresponding workstream date. Generally, medium-priority issues should be
+done within two weeks, high-priority within a week, and the highest-priority
+tickets within a few days.
+  >Note: In GitHub, you are allowed to set the due date only for a ticket within
+  >a project. Thus for stand alone tickets in GitHub, adjust the due date with
+  >labels of priority (high, low).
+* **Assignee**. Assign Alexander Paramonov or Denis Kryvchuk if the issue
```
Comment (dnillovna) [inline help/develop/how-to/report-tickets.md:76]: I think it is a good idea to add their nicknames or emails here

File: help/develop/admin/quality-assurance.md [hunk 2]
```diff
@@ -2,134 +2,172 @@
...
+[Docker images](https://hub.docker.com/u/datagrok).
+* Jenkins automates the process of building and testing code every time a change
+is made to the codebase in Git. It automatically executes the following tasks:
+  1. Static code analysis by [dartanalyzer](https://pub.dev/packages/analyzer)
+  2. Unit tests
+  3. Compilation
+  4. Packaging the platform in Docker containers
+  5. [Snyk](https://snyk.io/) and [Grype](https://github.com/anchore/grype/)
+  check for vulnerabilities
+* Docker Hub stores built artifacts, such as compiled code and packages, that
```
Comment (skalkin) [inline help/develop/admin/quality-assurance.md:45]: No comma needed before "that"

File: help/develop/admin/quality-assurance.md [hunk 3]
```diff
@@ -2,134 +2,172 @@
...
+and keep track of the results to ensure that the platform only gets better
+with time.
+* **Stress tests**. To ensure that the platform is stable under heavy load, we
+perform automated stress testing regularly. The real workflow is emulated by
+spawning many virtual machines that have Selenium installed on them and
+executing selected UI tests on them. This helps us identify steps responsible
+for the performance degradation.  For more details on different environments,
+tests, and interpretation of results, see
+[stress testing results](stress-testing-results.md).
+
```
Comment (skalkin) [inline help/develop/admin/quality-assurance.md:101]: I think automated package tests are missing

File: help/develop/admin/quality-assurance.md [hunk 4]
```diff
@@ -2,134 +2,172 @@
...
-
-To make sure the platform is stable under heavy load (either many users working simultaneously or executing
-CPU-intensive computations), we perform automated
-[stress testing](https://en.wikipedia.org/wiki/Stress_testing)
-regularly. The real workflow is emulated by spawning many virtual machines that have Selenium installed on them and
-executing selected [UI tests](#ui-tests) on them. There are two different passes.
+### Test manager
 
-In the first pass, an ordinary yet intensive user activity is emulated by random-ranging tests.
+Is a tool within the Datagrok platform that provides a convenient interface for
```
Comment (skalkin) [inline help/develop/admin/quality-assurance.md:154]: We shouldn't start paragraphs with "Is a tool" or "It is the". I think it should be "Test Manager is a tool ..."

File: help/develop/admin/quality-assurance.md [hunk 5]
```diff
@@ -2,134 +2,172 @@
...
 
-In the second pass, each of the selected tests is executed on all machines simultaneously. This helps us identify steps
-responsible for the performance degradation.
+### Usage Analysis
 
-For more details on different environments, tests, and interpretation of results, please
-see [stress testing results](stress-testing-results.md).
+It is the
+[Datagrok package](https://github.com/datagrok-ai/public/tree/master/packages/UsageAnalysis)
+designed to provide insights into how the Datagrok platform is utilized. Usage
```
Comment (skalkin) [inline help/develop/admin/quality-assurance.md:164]: Let's shorten the description, make it one sentence.

File: help/develop/how-to/report-tickets.md [hunk 6]
```diff
@@ -0,0 +1,92 @@
+---
+title: "Report bugs and feature requests"
+---
+
+This guide provides an easy set of rules for reporting issues and managing
```
Comment (skalkin) [inline help/develop/how-to/report-tickets.md:5]: Get rid of "easy", it's unnecessary and doesn't sound well, too (the author probably meant "simple") :)

File: help/develop/how-to/report-tickets.md [hunk 7]
```diff
@@ -0,0 +1,92 @@
...
+tickets in Datagrok. We use two issue tracking systems:
+
+* [JIRA](https://reddata.atlassian.net/)
+for internal issues integrated with BitBucket
+* [GitHub Tracker](https://github.com/datagrok-ai/public/issues)
+for all externally reported issues.
+
+We recommend reporting issues in JIRA if you have access to it, but if you
+don't, use GitHub Tracker. The rules of the bug report format adopted by the
+company do not differ in both issue tracking systems.
```
Comment (skalkin) [inline help/develop/how-to/report-tickets.md:15]: "are the same" would be better than "do not differ"

File: help/develop/how-to/report-tickets.md [hunk 8]
```diff
@@ -0,0 +1,92 @@
...
+  summary for an enhancement. If relevant, use a hierarchical representation of
+  the issue location: list modules and submodules in their logical order using
+  the “|” symbol.
+
+|Issue type|Formula|Example|
+|----------|-------|-------|
+|Bug report| [Platform module] or [Package name] : [Unexpected behavior]| Line chart: Y-axis label is broken|
+|New feature or Improvement| [Platform module] or [Package name] : [New feature name] or [Improvement summary]| Bar Chart: context menu harmonization|
+
+* **Description**. Is important for testing and writing release notes. For an
```
Comment (skalkin) [inline help/develop/how-to/report-tickets.md:38]: Do not start sentences with "is". But continuing the sentence like that is ok:

**Description** is important

**Review by skalkin — CHANGES_REQUESTED** [review]
Looks great, but let's address my comments first; just merge it after that.

**Pull request merged**